crates/configuration/src/parachain.rs

```diff
@@ -124,9 +136,16 @@ impl Default for ParachainConfigBuilder<Initial> {
         Self {
             config: ParachainConfig {
                 id: 100,
-                chain: None,
+                chain: ""
+                    .try_into()
+                    .expect("empty string should be valid. this is a bug, please report it: https://github.com/paritytech/zombienet-sdk/issues"),
```

Comment:
```suggestion
                    .expect("Empty string should be valid. This is a bug, please report it at: https://github.com/paritytech/zombienet-sdk/issues"),
```

Reply:
I think we should discuss about this, since if we allow to set an `empty` string as chain value we should handle that internally since will fail at running command like 
```
polkadot build-spec --chain
```

What are the `pros` on allow empty strings as valid `chain`?

Reply:
The config can't be built with an empty chain, this is only to fullfill the initial values but the typestate prevent to have non defined chain, or at least it should be added at validation, atm we only have no spaces validation

Reply:
Ahh oks, thanks for the clarification. Also, I think we should add the check into validation step 👍 .

Thanks! (I will mark as resolved)